docs/rules/prefer-user-event.md

```diff
@@ -0,0 +1,126 @@
+| `fireEvent.mouseEnter()`   | <ul><li>`userEvent.hover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                                                         |
+| `fireEvent.mouseLeave()`   | <ul><li>`userEvent.unhover()`</li></ul>                                                                                                                                 |
+| `fireEvent.mouseMove()`    | <ul><li>`userEvent.hover()`</li><li>`userEvent.unhover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                           |
+| `fireEvent.mouseOut()`     | <ul><li>`userEvent.unhover()`</li></ul>                                                                                                                                 |
+| `fireEvent.mouseOver()`    | <ul><li>`userEvent.hover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                                                         |
+| `fireEvent.mouseUp()`      | <ul><li>`userEvent.click()`</li><li>`userEvent.dblClick()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                          |
+| `fireEvent.paste()`        | <ul><li>`userEvent.paste()`</li></ul>                                                                                                                                   |
+| `fireEvent.pointerDown()`  | <ul><li>`userEvent.click()`</li><li>`userEvent.dblClick()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                          |
+| `fireEvent.pointerEnter()` | <ul><li>`userEvent.hover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                                                         |
+| `fireEvent.pointerLeave()` | <ul><li>`userEvent.unhover()`</li></ul>                                                                                                                                 |
+| `fireEvent.pointerMove()`  | <ul><li>`userEvent.hover()`</li><li>`userEvent.unhover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                           |
+| `fireEvent.pointerOut()`   | <ul><li>`userEvent.unhover()`</li></ul>                                                                                                                                 |
+| `fireEvent.pointerOver()`  | <ul><li>`userEvent.hover()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                                                         |
+| `fireEvent.pointerUp()`    | <ul><li>`userEvent.click()`</li><li>`userEvent.dblClick()`</li><li>`userEvent.selectOptions()`</li><li>`userEvent.deselectOptions()`</li></ul>                          |
```

Comment:
I don't think we need to repeat `fireEvent`/`userEvent` and parentheses in this table.